### ISSUE TITLE
`domainRegex` does not match k8s or docker DNS names

### ISSUE THREAD

**Issue by Schnitzel** [aaronbarnardsound/lnmessage#35]
in Docker or Kubernetes a DNS name can just be a single name, an example for `docker-compose.yml`:

```
services:
  core-lightning:
    build:
      context: .
      dockerfile: Dockerfile

  core-lightning-application:
    build:
      context: .
      dockerfile: core-lightning-application.dockerfile
```

will allow you to resolve just `core-lightning` inside the `core-lightning-application` container, example:

```# dig core-lightning

; <<>> DiG 9.16.42-Debian <<>> core-lightning
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 48634
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 0

;; QUESTION SECTION:
;core-lightning.			IN	A

;; ANSWER SECTION:
core-lightning.		600	IN	A	172.18.0.6

;; Query time: 2 msec
;; SERVER: 127.0.0.11#53(127.0.0.11)
;; WHEN: Fri Jul 21 18:26:06 UTC 2023
;; MSG SIZE  rcvd: 62
```

but unfortunately `lnmessage` does not like just `core-lightning`, there is an error:

```core-lightning-application_1  | > cln-application@0.0.3 start
core-lightning-application_1  | > npm run serve -w cln-application-backend
core-lightning-application_1  |
core-lightning-application_1  |
core-lightning-application_1  | > cln-application-backend@0.0.3 serve
core-lightning-application_1  | > node ./dist/server.js
core-lightning-application_1  |
core-lightning-application_1  | {"level":"error","message":"\tFailed to read rune for Commando connection: {}","timestamp":"2023-07-21 18:24:55:2455"}
core-lightning-application_1  | file:///app/node_modules/lnmessage/dist/validation.js:10
core-lightning-application_1  |         throw new Error(`${ip} is not a valid IP or DNS address`);
core-lightning-application_1  |               ^
core-lightning-application_1  |
core-lightning-application_1  | Error: core-lightning is not a valid IP or DNS address
core-lightning-application_1  |     at validateInit (file:///app/node_modules/lnmessage/dist/validation.js:10:15)
core-lightning-application_1  |     at new LnMessage (file:///app/node_modules/lnmessage/dist/index.js:71:9)
core-lightning-application_1  |     at new LightningService (file:///app/apps/backend/dist/service/lightning.service.js:20:34)
core-lightning-application_1  |     at file:///app/apps/backend/dist/service/lightning.service.js:55:26
core-lightning-application_1  |     at ModuleJob.run (node:internal/modules/esm/module_job:194:25)
core-lightning-application_1  |
core-lightning-application_1  | Node.js v18.16.0
core-lightning-application_1  | npm ERR! Lifecycle script `serve` failed with error:
core-lightning-application_1  | npm ERR! Error: command failed
core-lightning-application_1  | npm ERR!   in workspace: cln-application-backend@0.0.3
core-lightning-application_1  | npm ERR!   at location: /app/apps/backend```

**Comment by lnbc1QWFyb24** [comment]
Ah yep, i'll get that regex updated to handle all valid dns names.

**Comment by lnbc1QWFyb24** [comment]
Fixed in #36 

Version `0.2.3` has been published, so let me know if looks good on your end!
